src/api/admin/mermaid.py

```diff
@@ -1010,3 +1010,67 @@ class CovariateAdmin(BaseAdmin):
 @admin.register(AuditRecord)
 class AuditRecordAdmin(BaseAdmin):
     list_display = ("event_on", "event_type", "model", "record_id")
+
+
+class ObsBenthicPhotoQuadratInline(ObservationInline):
+    model = ObsBenthicPhotoQuadrat
+    cache_fields = ["attribute", "growth_form"]
+
+
+@admin.register(BenthicTransectPhotoQuadrat)
+class BenthicTransectPhotoQuadratAdmin(BaseAdmin):
+    list_display = ("name", "quadrat_size", "depth")
+    inlines = (
+        ObserverInline,
+        ObsBenthicPhotoQuadratInline,
+    )
+    readonly_fields = ["created_by", "updated_by", "cr_id"]
+    search_fields = [
+        "transect_quadrat__sample_event__site__name",
+        "transect_quadrat__sample_event__sample_date",
+        "transect_quadrat__sample_event__site__project__name",
+    ]
+    ordering = ["transect_quadrat__sample_event__site__name"]
+
+    def name(self, obj):
+        return str(obj.quadrat)
+
+    name.admin_order_field = "transect_quadrat__sample_event__site__name"
+
+    def quadrat_size(self, obj):
+        return obj.transect_quadrat.quadrat_size
+
+    def depth(self, obj):

```

Comment:
depth `admin_order_field`?

Reply:
you want depth instead of site name for ordering?